contracts/colonyNetwork/ColonyNetworkExtensions.sol

```diff
@@ -0,0 +1,128 @@
+/*
+  This file is part of The Colony Network.
+
+  The Colony Network is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity 0.5.8;
+pragma experimental ABIEncoderV2;
+
+import "../colony/ColonyDataTypes.sol";
+import "../colonyNetwork/IColonyNetwork.sol";
+import "../extensions/ColonyExtension.sol";
+import "./ColonyNetworkStorage.sol";
+
+
+contract ColonyNetworkExtensions is ColonyNetworkStorage {
+
+  // Public functions
+
+  function addExtension(bytes32 _extensionId, address _resolver)
```

Comment:
Should there be a `removeExtension` function? I haven't thought through potential consequences...

Reply:
I don't feel strongly, I'd say no need for now